migrations/versions/0140_sms_prefix_non_nullable.py

```diff
@@ -29,9 +29,23 @@ def upgrade():
         nullable=False,
     )
 
+    op.alter_column(
+        'services_history',
+        'prefix_sms',
+        existing_type=sa.BOOLEAN(),
+        nullable=False,
+    )
+
 
 def downgrade():
 
+    op.alter_column(
+        'services_history',
+        'prefix_sms',
+        existing_type=sa.BOOLEAN(),
+        nullable=True,
+    )
+
```

Comment:
Modifying old migrations seems wrong? Is that allowed? I think Dave tried that and hit huge problems?